### ISSUE TITLE
Element 'nav' is not supported with html5 custom_definition

### ISSUE THREAD

**Issue by sonicmolson** [mewebstudio/Purifier#63]
Using the default configuration using vendor publish, when I add "nav" to the default HTML.allowed setting, like so
`'HTML.Allowed' => 'div,b,strong,i,em,u,a[href|title],ul,ol,li,p[style],br,span[style],img[width|height|alt|src],nav'1`

 I get the following error:
`Element 'nav' is not supported`
`in HTMLDefinition.php (line 311)`

I thought that the custom_definition should have resolved that. Is there something I have to do after to enable the HTML

I'm running mews/purfier 2.0.7 with htmlpurifer 4.8.0

**Comment by ty666** [comment]
I have the same problem！

**Comment by brucebnu** [comment]
yeah ! I have the same problem too！I need add "video" tags and something attributes.

**Comment by ty666** [comment]
@brucebnu https://github.com/mewebstudio/Purifier/pull/52#issuecomment-343273597
